packages/bee-q/src/components/panel/bq-panel.type.tsx

```diff
@@ -0,0 +1,16 @@
+import { FloatingUIPlacement } from '../../services/interfaces';
+
+export const PANEL_PLACEMENT: ReadonlyArray<FloatingUIPlacement> = [
+  'top',
+  'top-start',
+  'top-end',
+  'right',
+  'right-start',
+  'right-end',
+  'bottom',
+  'bottom-start',
+  'bottom-end',
+  'left',
+  'left-start',
+  'left-end',
+] as const;
```

Comment:
Back to the need for PANEL_PLACEMENT, I see it is only used in the Storybook doc example. In that case, I would rather prefer to hardcode the value in the *.stories.tsx file than adding a useless type:

```js
  argTypes: {
    distance: { control: 'number' },
    placement: {
      control: { type: 'select' },
      options: [
        'top',
        'top-start',
        'top-end',
        'bottom',
        'bottom-start',
        'bottom-end',
        'right',
        'right-start',
        'right-end',
        'left',
        'left-start',
        'left-end',
      ],
    }
```